### ISSUE TITLE
Dencki

### ISSUE THREAD

**Issue by Dencki** [ZechBron/zVirus-Gen#2]
who tested it working?  Or is it just an apk file?

**Comment by ZechBron** [comment]
@Dencki It's a  collection of apk viruses that you can send to your victim's through download link.

**Comment by Dencki** [comment]
> @Dencki It's a  collection of apk viruses that you can send to your victim's through download link.

but does it work?  otherwise I'm afraid to check

**Comment by DIEGO5136** [comment]
Yes, it works

Em qui, 20 de mai de 2021 11:20, Dencki ***@***.***> escreveu:

**Comment by ZechBron** [comment]
@DIEGO5136 Yes it will. A download link will appear and give it to your victim and ask your victim to install the apk file

**Comment by vivekviroy** [comment]
Ok then what can i do after sending this virus to victims phone?
Can i gather information by thi svirus from victim's phone ?
Tell me fast

**Comment by DIEGO5136** [comment]
I do not know. I didn't create the virus

Em qui, 10 de jun de 2021 23:54, vivekviroy ***@***.***>
escreveu:

**Comment by Umut989** [comment]
Where does the link write
